pages/common/git-reauthor.md

```diff
@@ -0,0 +1,17 @@
+# git reauthor
+
+> Replace details about an author identity, since this command rewrites git's history --force will be needed when `git push` is used.
```

Comment:
```suggestion
> Change details about an author identity. Since this command rewrites the Git history, `--force` will be needed when pushing next time.
```